ckanext/vitality_prototype/plugin.py

```diff
@@ -362,6 +361,19 @@ def after_search(self, search_results, search_params):
 
         # However, at a time only loads a portion of the results
         datasets = search_results['results']
+        
+        # Checks if the search requires restricted variable checking
+        # TODO Overall costly - find a better alternative
+        restricted_search_enabled = False
+        restricted_search_eovs = []
+        restricted_search_keywords = []
+        for x in search_params['fq'][0].replace(")","").replace("(", "").split(" "):
+            if(x.startswith('res_extras_eov_restricted')):
+                restricted_search_eovs.append(x.split('"')[1])
+                restricted_search_enabled = True
```

Comment:
restricted_search_eovs = [x.split('"')[1] for x in search_params['fq'][0].replace (...) if x in search_params['fq'][0]]